src/machines/java/com/enderio/machines/common/blockentity/base/PoweredTaskMachineEntity.java

```diff
@@ -1,88 +1,52 @@
 package com.enderio.machines.common.blockentity.base;
 
-import com.enderio.api.UseOnly;
 import com.enderio.api.capacitor.ICapacitorScalable;
 import com.enderio.api.io.energy.EnergyIOMode;
-import com.enderio.core.common.sync.FloatDataSlot;
-import com.enderio.core.common.sync.SyncMode;
 import com.enderio.machines.common.block.ProgressMachineBlock;
 import com.enderio.machines.common.blockentity.task.PoweredTask;
+import com.enderio.machines.common.blockentity.task.host.MachineTaskHost;
 import net.minecraft.core.BlockPos;
 import net.minecraft.nbt.CompoundTag;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.entity.BlockEntityType;
 import net.minecraft.world.level.block.state.BlockState;
-import net.minecraftforge.fml.LogicalSide;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Generic class for a machine that can perform a task.
  */
 public abstract class PoweredTaskMachineEntity<T extends PoweredTask> extends PoweredMachineEntity {
```

Comment:
This class has just been "stubbed" to use the new task host, it prove that the host had the same functionality as the original.
The idea however is for this class (and other abstract classes inheriting from PoweredMachineEntity) will be getting removed.